ci-operator/config/maistra/istio/maistra-istio-maistra-2.3.yaml

```diff
@@ -1,18 +1,28 @@
+base_images:
+  cli:
+    name: "4.10"
```

Comment:
Does it make sense use ocp 4.11 as it's the latest?

Reply:
From https://docs.ci.openshift.org/docs/how-tos/cluster-claim/#existing-cluster-pools
It looks like there are more standby or ready OCP 4.10 clusters than OCP 4.11 ones.
That means if we use ocp 4.11 we may need to wait more time in line when no standby cluster in the pool.